ProjectSettings/InputManager.asset

```diff
@@ -268,7 +268,7 @@ InputManager:
     negativeButton: 
     positiveButton: enter
     altNegativeButton: 
-    altPositiveButton: space
+    altPositiveButton: 
```

Comment:
Removed space as an option from the `Submit` action. It was causing the map to open/close every time the player jumped 😂